modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -297,14 +295,6 @@ def mpi_veteran
       )
     end
 
-    def email_identifier
-      return { id_type: 'email', id_value: email } if email.present?
-
-      icn = mpi_veteran.mpi_icn
-
-      { id_type: 'ICN', id_value: icn } if icn.present?
-    end
-
```

Comment:
I moved this method to the public section because it contains the exact information we need in the appeal received job. The old version of the job expected this information to be passed in via the `email_identifier` option. Making this publicly accessible allows us to call it on the appeal directly in the job instead of passing it in.